INSTALL.md

```diff
@@ -31,6 +34,15 @@ This document provides a guide for installing AWX.
 
 ## Getting started
 
+### Minimum Hardware Requirements
+
+#### CPU
+2 CPUs recommended.
+
+#### RAM
+2 GB RAM is the minimum .
```

Comment:
Extra space prior to the period.

Reply:
@leifmadsen Why should there be an extra space prior to the period here but not at the " 4GB RAM is recommended." line?

Reply:
@KnechtionsCoding that's exactly what I'm pointing out. It's inconsistent. The space prior to the period on this line should be removed.

Reply:
@leifmadsen Sorry, I clearly shouldn't comment after a long day. I'll fix that so it is consistent now.